install-docker.yml

```diff
@@ -1,26 +1,25 @@
---- 
-- hosts: all
-  remote_user: ubuntu
-  become: yes
-
+---
+- name: Find out which python version is avaible
+  hosts: all
+  gather_facts: false
+  remote_user: "{{ username }}"
```

Comment:
See previous comment. You could just omit this line and use `ansible-playbook -u username`